build/.azure-pipelines.TemplateValidation.yml

```diff
@@ -8,45 +8,45 @@ jobs:
     matrix:
       DefaultArguments:
         templateArgs: ''
-      Blank:
-        templateArgs: '-preset blank'
+      Recommended:
+        templateArgs: '-preset recommended'
       SkiaOnlyHeads:
         templateArgs: '-platforms gtk wpf linux-fb'
       MobileOnlyHeads:
         templateArgs: '-platforms android ios maccatalyst'
-      BlankMarkup:
-        templateArgs: '-preset blank -markup csharp'
-      BlankNoCpm:
-        templateArgs: '-preset blank --cpm false'
+      RecommendedMarkup:
+        templateArgs: '-preset recommended -markup csharp'
```

Comment:
Can we reorder so that the validation pairs are adjacent (ie move "--cpm false" and "-preset recommended --cpm false" so that they're adjacent). This will help with long term maintenance of the validation list